torchao/quantization/quantize_/common/packing_format.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+from enum import Enum
+
+
+# can switch to StrEnum (https://docs.python.org/3/library/enum.html#enum.StrEnum)
+# after python 3.10 is end of life (https://devguide.python.org/versions/)
+class PackingFormat(str, Enum):
+    """Packing format for Tensor subclasses in torchao, enum for how
```

Comment:
for "tensor subclass", or for "individual plain tensor attribute of a tensor subclass"?

Reply:
I think both applies since tensor subclass is structured as derived dtype + format, but I can use the "packing format for quantized data" to be more specific